src/components/DisplayForm/DisplayForm.js

```diff
@@ -0,0 +1,82 @@
+export default function DisplayForm({
+  name,
+  bio,
+  age,
+  breed,
+  image,
+  welcome,
+  message,
+  upDog,
+  submitText,
+  handleSubmit,
+}) {
+  return (
+    <div>
+      <p>
+        {welcome}
+        {name}
```

Comment:
I might have something more like `{name && name}` just to make it more clear that this is only sometimes used